pwndbg/heap/structs.py

```diff
@@ -796,8 +916,10 @@ class MallocPar(CStruct2GDB):
         _c_struct = c_malloc_par_2_26
     elif pwndbg.glibc.get_version() >= (2, 24):
         _c_struct = c_malloc_par_2_24
-    else:
+    elif pwndbg.glibc.get_version() >= (2, 15):
```

Comment:
Nitpick: if we do this change, can we call to get_version() once and save it in a `_version` variable or sth?